packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -20,27 +24,20 @@ class Product extends Component {
             root: string
         }),
         item: shape({
-            name: string,
-            options: arrayOf(
-                shape({
-                    name: string,
-                    value: string
-                })
-            ),
-            price: shape({
-                regularPrice: shape({
-                    amount: shape({
-                        currency: string.isRequired,
-                        value: number.isRequired
-                    }).isRequired
-                }).isRequired
-            }).isRequired
-        })
+            item_id: number.isRequired,
+            name: string.isRequired,
+            price: number.isRequired,
+            product_type: string,
+            qty: number.isRequired,
+            quote_id: string,
+            sku: string.isRequired
+        }),
```

Comment:
`item` should be `isRequired`, since we're accessing properties on it without any safety checks